zerver/lib/markdown/__init__.py

```diff
@@ -1762,8 +1762,8 @@ def prepare_realm_pattern(source: str) -> str:
 
 # Given a regular expression pattern, linkifies groups that match it
 # using the provided format string to construct the URL.
-class RealmFilterPattern(markdown.inlinepatterns.Pattern):
-    """ Applied a given realm filter to the input """
+class RealmLinkifierPattern(markdown.inlinepatterns.Pattern):
```

Comment:
LinkifierPattern would be a better name, I think, especially since our plans include making Stream-level ones possible as well?

Reply:
Yep! Renamed to `LinkifierPattern`.